internal/isoeditor/rhcos.go

```diff
@@ -237,22 +64,17 @@ func RamdiskImageArchive(netFiles []staticnetworkconfig.StaticNetworkConfigData,
 		return nil, err
 	}
 
-	return getCompressedArchive(buffer)
-}
-
-func addCustomRAMDisk(clusterISOPath string, netFiles []staticnetworkconfig.StaticNetworkConfigData, clusterProxyInfo *ClusterProxyInfo, ramdiskOffsetInfo *OffsetInfo) error {
-	compressedArchive, err := RamdiskImageArchive(netFiles, clusterProxyInfo)
-	if err != nil {
-		return err
+	// Run gzip compression
```

Comment:
Just making sure, this is coming to replace the functionality you removed with `getCompressedArchive()`, right?

Reply:
Yup, the function would have only been called from this one place after the changes so I just inlined it here.